src/haxelib/client/Main.hx

```diff
@@ -173,8 +173,8 @@ class Main {
 		addCommand("local", local, "install the specified package locally", Development, false);
 		addCommand("dev", dev, "set the development directory for a given library", Development, false);
 		//TODO: generate command about VCS by Vcs.getAll()
-		addCommand("git", function()doVcs(VcsID.Git), "use Git repository as library", Development);
-		addCommand("hg", function()doVcs(VcsID.Hg), "use Mercurial (hg) repository as library", Development);
+		addCommand("git", function() doVcs(VcsID.Git), "use Git repository as library", Development);
+		addCommand("hg", function() doVcs(VcsID.Hg), "use Mercurial (hg) repository as library", Development);
```

Comment:
Modified it and then returned it back, thought the spaces where from before, I'll return it to normal in the squash.